### ISSUE TITLE
Character mouth overly sensitive to "cold"

### ISSUE THREAD

**Issue by PatrikLundell** [CleverRaven/Cataclysm-DDA#59395]
### Describe the bug

My character is currently shown as "chilly (mouth)" (-30)  at a temperature of 4 degrees Celsius, and I've previously gotten the "frostbite in a few hours" warning at 2 degrees Celsius.

You should be able to deal with an exposed mouth a bit below zero (and quite a fair bit if you're resistant to cold) unless you have a cold allergy, and frostbite isn't even physically possible above freezing, as it's caused by formation of ice crystals in the tissue (and those don't contain pure water, so the freezing temperature is lower still).

You can get hypothermia from above freezing temperatures, as that's caused by lowering of the body temperature due to an inability to retain enough heat to keep the temperature steady, but that's a different issue.

### Steps to reproduce

1. Make a player with an exposed face.
2. Adjust time to mid/late autumn and find a temperature of 4 degrees Celsius.
3. Walk around for a bit (don't know for how long).
4. Find that your character has a cold mouth.

### Expected behavior

The mouth should be more resistant to cold than the rest of the body (except the eyes), with the hands probably more resistant than the remainder of the body.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19043.1826 (21H1)
- Game Version: d6a1777 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]
+ JSON "mod" to disable portal storms
+ JSON addition of welding supply crafting to workshop expansion
+ JSON addition of tallow recipe to canteen expansion

**Comment by Maleclypse** [comment]
@deafmole do you want to be added to the approved bug reporter list for the bot?

**Comment by PatrikLundell** [comment]
In my case the wind was light, so there wasn't any wind chill factor.

**Comment by Gotdamnmiracle** [comment]
> In my case the wind was light, so there wasn't any wind chill factor.

Yeah. I don't think this is (yet) factored into the game other than functionally reducing bodypart temp and causing frostbite. Which is sort of a crude representation of windburn, I guess. Out of scope for this PR though.

**Comment by Terrorforge** [comment]
Isn't "chilly" just the "uncomfortable" level, or does it cause real harm? I'm struggling to remember what 4 degrees on your face feels like here in the midst of summer, but it seems reasonable do describe it as "chilly", no? I'm also not sure how accurate those frostbite warnings are. I've definitely gotten the warnings at relatively warm temperatures, but I'm not sure I've ever actually gotten frostbite.

Because I don't think the mouth is actually more resistant to cold than any other body part. I think we just tend to accept the fact that it gets uncomfortable because the alternatives are even more annoying, and the game models that fine.

One thing it doesn't model though is that hoods keep your face warm without covering it, if nothing else because it keeps the wind off. There is a COLLAR tag that does this, but it's largely absent from items that also have the HOOD tag, even ones that would realistically have both hoods _and_ collars

**Comment by PatrikLundell** [comment]
"Chilly" is indeed the uncomfortable value, but given that it turns into a "harm is on the way" at just two degrees lower, there isn't much of a margin (I believe you can stay in a 6 degree basement naked without being uncomfortable, something I definitely wouldn't do in real life).

There was an error report recently by someone who tried to craft clothing to protect against cold at 2 degrees and died because "frostbite" destroyed the arms, slowing crafting, and eventually killed the character completely. I believe that character had standard starting clothing with a standard start.

I do believe exposed body parts get more resistant to abuse over time, provided harm isn't caused along the way.

**Comment by Gotdamnmiracle** [comment]
> Does two degrees drop a naked body part all the way from -30 to -60, or are the frostbite mechanics so decoupled from the warmth mechanics that you can die despite only being kind of chilly? I think that might be the real problem

Anecdotally, I have seen some really funky shifts similar to this if something is uncovered (or in the other direction if covered while it's warm out), but I cannot confirm if it's this scuffed.

I do think that the abstractions mess things up pretty bad, however. I really wish the numbers on bodyparts were the temperature the bodypart is exposed to (ambient + windchill). Why is it abstracted again?

**Comment by Terrorforge** [comment]
Unclear. Nobody who's currently with the project likes the temperature mechanics or understands why they're measured, as one memorable comment put it, "in pixie dust per square rainbow".

**Comment by Gotdamnmiracle** [comment]
Well that's going to be a nightmare to fix the clothing once all the breathability, coverage, and materials are all tweaked to AGAIN go in and make it conform to a sane temperature system.

**Comment by deafmole** [comment]
@Maleclypse Yeah, I'd appreciate it.
